cds-snc/report-a-cybercrime
# Hookup feedback to notify

# Description

email feedback (to Mel!) using Notify

To test:
1) checkout branch locally `git checkout hookup-feedback-to-notify`
2) add environment variables to your .env
```
NOTIFY_FEEDBACK_TEMPLATE_ID= <get this from me>
FEEDBACK_EMAIL= <your email address>
```
3) `npm run build && node server.js`
4) go to `http://localhost:3000/finalFeedback`, fill out and submit
5) you should get an email of the feedback (in json format)

# Checklist:


# Required followup work

We should probably hook into the work CDS did on gathering feedback into a prober database.
